src/sentry/static/sentry/app/components/createTeam/createTeamForm.jsx

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {addSuccessMessage} from '../../actionCreators/indicator';
+import {t, tct} from '../../locale';
+import Form from '../../views/settings/components/forms/form';
+import SentryTypes from '../../proptypes';
+import TextField from '../../views/settings/components/forms/textField';
+
+export default class CreateTeamForm extends React.Component {
+  static propTypes = {
+    organization: SentryTypes.Organization.isRequired,
+    onSuccess: PropTypes.func.isRequired,
+    onSubmit: PropTypes.func,
+  };
+
+  handleCreateTeamSuccess = data => {
+    addSuccessMessage(tct('Added team [team]', {team: `#${data.slug}`}));
+    this.props.onSuccess(data);
+  };
+
+  render() {
+    let {organization} = this.props;
+    let features = new Set(organization.features);
+
+    return (
+      <React.Fragment>
+        <p>
+          {t(
+            "Teams group members' access to a specific focus, e.g. a major product or application that may have sub-projects."
```

Comment:
is this copy pasted? it seems a bit outdated. 
maybe you could say 
"Teams group members' access to specific projects, and can be assigned to issues"